### PR TITLE
refactor(sbt): Reimplement `parseIndexDir` usage

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — APPROVED** [review]

File: lib/modules/datasource/sbt-plugin/index.spec.ts [hunk 1]
```diff
@@ -92,10 +105,11 @@ describe('modules/datasource/sbt-plugin/index', () => {
         .get('/maven2/org/foundweekends/sbt-bintray/scala_2.12/sbt_1.0/')
         .reply(
           200,
-          html`
+          codeBlock`
```
Comment (viceice) [inline lib/modules/datasource/sbt-plugin/index.spec.ts:108]: why this change?
